app/src/main/java/org/wikipedia/main/MainFragment.kt

```diff
@@ -441,7 +440,7 @@ class MainFragment : Fragment(), BackPressedHandler, MenuProvider, FeedFragment.
 
     override fun usernameClick() {
         val pageTitle = PageTitle(UserAliasData.valueFor(WikipediaApp.instance.languageState.appLanguageCode) + ":" + AccountUtil.userName, WikipediaApp.instance.wikiSite)
-        UriUtil.visitInExternalBrowser(requireContext(), Uri.parse(pageTitle.uri))
+        CustomTabsUtil.openInCustomTab(requireContext(), pageTitle.uri)
```

Comment:
I think the intention is to open the user page in the native PageActivity (`PageActivity.newIntent(...)`) rather than as an external web page.

Reply:
Done